src/main/kotlin/kweb/html/fileUpload/FileFormInput.kt

```diff
@@ -21,8 +21,8 @@ class FileFormInput {
         this._inputElement = e
     }
 
-    fun setAccept(acceptedTypes: String): Unit = inputElement.execute("${inputElement.jsExpression}.accept=\"$acceptedTypes\";")
-    fun isMultiSelect(isMultiple: Boolean): Unit = inputElement.execute("${inputElement.jsExpression}.multiple=\"$isMultiple\";")
+    fun setAccept(acceptedTypes: String): Unit = inputElement.callJs("${inputElement.jsExpression}.accept=\"$acceptedTypes\";")
```

Comment:
Something like `callJs("document.getElementById({}).accept = {}", inputElement.id, acceptedTypes)`?
Because that should work. I was just so focused on changing all the instances of execute to callJs, that I overlooked adding our new variable syntax. I'll go back through and fix these this evening.